<launch>
    <!-- <param name="rosout_filename" value="/home/lifeipeng/testdata/log.log"/> -->
    <!-- <param name="rosout_screen" value="false"/> -->
    <param name="lidar_topic" value="/hesai/pandar"/>
    <param name="pub_points_topic" value="/velodyne_points"/>
    <param name="pub_packets_topic" value="/velodyne_packets"/>

    <node pkg="hesai_to_velodyne" type="pandar40p_to_velodyne" name="pandar40p_to_velodyne" args="XYZIRT XYZIRT"/>
    <!--node pkg="nodelet" type="nodelet" name="cloud_nodelet" output="screen" args="load velodyne_laserscan/LaserScanNodelet velodyne_nodelet_manager"/-->
    <node pkg="nodelet" type="nodelet" name="cloud_nodelet" output="screen" args="load velodyne_pointcloud/CloudNodelet velodyne_nodelet_manager">

    <node pkg="rviz" type="rviz" name="rviz_pandar40p_to_velodyne" required="true" args="-d $(find hesai_to_velodyne)/rviz/rviz.rviz"/>
</launch>
